### PR TITLE
[AdminBundle][Feature] Resolve all exceptions

### PR DESCRIPTION
**Pull request by cv65kr**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #1762

### CODE REVIEW

File: src/Kunstmaan/AdminBundle/Controller/ExceptionController.php [hunk 1]
```diff
@@ -29,8 +29,53 @@ public function indexAction(Request $request)
...
+     *
+     * @return RedirectResponse
+     *
+     * @throws \Doctrine\ORM\ORMInvalidArgumentException
+     * @throws \Doctrine\ORM\OptimisticLockException
+     * @throws \InvalidArgumentException
+     */
+    public function resolveAllAction()
+    {
+        /* @var EntityManager $em */
```
Comment (sandergo90) [inline src/Kunstmaan/AdminBundle/Controller/ExceptionController.php]: EntityManagerInterface is better here

File: src/Kunstmaan/AdminBundle/Controller/ExceptionController.php [hunk 2]
```diff
@@ -29,8 +29,53 @@ public function indexAction(Request $request)
...
+        $em = $this->getEntityManager();
+
+        $this->getAdminListConfigurator();
+
+        $exceptions = $em->getRepository(Exception::class)->findAllNotResolved();
```
Comment (sandergo90) [inline src/Kunstmaan/AdminBundle/Controller/ExceptionController.php]: It's maybe better to create an update query for this. Right now you are loading all Exception entities inside the entitymanager.

File: src/Kunstmaan/AdminBundle/Controller/ExceptionController.php [hunk 3]
```diff
@@ -29,8 +29,53 @@ public function indexAction(Request $request)
...
+     * @throws \Doctrine\ORM\ORMInvalidArgumentException
+     * @throws \Doctrine\ORM\OptimisticLockException
+     * @throws \InvalidArgumentException
+     */
+    public function resolveAllAction()
+    {
+        /* @var EntityManager $em */
+        $em = $this->getEntityManager();
+
+        $this->getAdminListConfigurator();
```
Comment (sandergo90) [inline src/Kunstmaan/AdminBundle/Controller/ExceptionController.php]: Can be moved to the $indexUrl = $this->getAdminListConfigurator()->getIndexUrl();

File: src/Kunstmaan/AdminBundle/Repository/ExceptionRepository.php [hunk 4]
```diff
@@ -16,6 +16,14 @@ public function findAllHigherThanDays(\DateTimeInterface $date)
             ->getResult();
     }
 
+    public function findAllNotResolved()
```
Comment (sandergo90) [inline src/Kunstmaan/AdminBundle/Repository/ExceptionRepository.php]: Add docblocks

**Comment by cv65kr** [comment]
@sandergo90, can You review again?

**Pull request merged**